README.md

```diff
@@ -1,16 +1,41 @@
 # DevOps-Apprenticeship-Notes
 
 
+
 * Take a look at our Bitwise Industries Standards and Practices for example markdown and best practices: https://github.com/Shift3/standards-and-practices
 
 ## Contributing to this Repository
 
+
 #### You are expected to contribute _something_ to our processes. You can do this in many different ways, such as addng documentation that helped you out, writing up a markdown sheet for this repository on a topic you are passionate about, leading a workshop, or posting discussion topics in Bitwise's #devops-apprenticeship-attachment channel.
 
 
 
 #### [Click here](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) to get started!
 
+# GITHUB
```

Comment:
Do you mind changing the size of the headers to double ## and the sub headers to triple ### ? This will help with consistency and simplify navigating the README.